plugins/optimization-detective/class-od-html-tag-processor.php

```diff
@@ -338,15 +352,30 @@ public function next_token(): bool {
 		return true;
 	}
 
+	/**
+	 * Gets the number of times the cursor has moved.
+	 *
+	 * @since n.e.x.t
+	 * @see self::next_token()
+	 * @see self::seek()
+	 *
+	 * @return int Count of times the cursor has moved.
+	 */
+	public function get_cursor_move_count(): int {
+		return $this->cursor_move_count;
+	}
+
 	/**
 	 * Gets the number of times next_token() was called.
 	 *
 	 * @since 0.4.1
 	 * @see self::next_token()
+	 * @deprecated Use {@see self::get_cursor_move_count()} instead.
```

Comment:
Are we deprecating this so dependent plugins don't break? If we don't explicitly call from outside the plugin, couldn't we just remove this?

Reply:
Yes, we could just remove it. I'm happy to do do if you think that's best. It's extremely unlikely anyone is using these deprecated methods.

Reply:
Removed in f4d0f8c9